### DISCUSSION TITLE
流萤老婆真爱我

### DISCUSSION THREAD

**Discussion by IMOUYO** [share121/inter-knot#408] [其他]
![屏幕截图 2024-06-30 133926](https://github.com/user-attachments/assets/0e0b1626-7b99-473c-929e-9fb9fbbb9b4c)
![屏幕截图 2024-06-30 133926](https://github.com/user-attachments/assets/0e0b1626-7b99-473c-929e-9fb9fbbb9b4c)
100抽流萤老婆和阮梅都来力，因为已经被室友真实了，不敢发BB圈，随发到这里，这回他没办法真实我了哈哈😆😆😆

**Comment by Kamisato100** [comment]
![OQC8DMPK8YY (7 ZS07LFE7](https://github.com/user-attachments/assets/45c3aebc-dd5a-4142-bf3a-59b94a77e36e)
![Y~ PH(RIRL8@KU$}TQQ1@$4](https://github.com/user-attachments/assets/84c3faa4-0cf7-4406-85be-a50b850c3756)

**Comment by TTGFCCt** [comment]
![Image_1723231873809](https://github.com/user-attachments/assets/88200fa1-cf3a-4882-8c1b-1058291546d4)
我感觉你挺欧的

**Comment by 117518** [comment]
流萤小保底出的，但是一颗星穹都没有了（泪目）
